testing/test-scripts/helloworld.sh

```diff
@@ -51,26 +53,32 @@ MAX_LOOP=5
 SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 A8CLI=$SCRIPTDIR/../../bin/a8ctl-beta-linux
 
+. $SCRIPTDIR/testing-common.sh
+
 # On single runs, have seen the routing to not be 100% following the
 # rule set.  So will loop to retry a few times to see if can get a
 # passing test.
-# TODO: add k8s TPR based testing
 retry_count=1
 while [  $retry_count -le $((MAX_LOOP)) ]; do
     ############# Set/Verify setting of default route #############
 
     # Clean up the routing rules from the previous testcase
-    $A8CLI rule-delete -a -f
+    cleanup_all_rules
 
     echo ""
     echo "Set/verify the Hello World default route to v1"
-    echo "  Invoking a8ctl to set the default route"
-    $A8CLI rule-create -f $SCRIPTDIR/helloworld-default-route-rules.json > /tmp/a8ctl_output.txt
+    if [ "$ENV" == "docker" ]; then
+        create_rule $SCRIPTDIR/helloworld-default-route-rules.json
+    else
+        create_rule $SCRIPTDIR/helloworld-default-route-rules.yaml
```

Comment:
Also, seems like controlplane and demo topologies are copied to examples using a `docker` or `k8s` prefix, but rules are copied as is. In case there is ever a divergence, we should consider adding a prefix to the rules on copying as well.